### PR TITLE
[Language] Init Vietnamese Translate Done 100%

### PR DESCRIPTION
**Pull request by dungphanxuan**
Add Vietnamese Language

image

### CODE REVIEW

**Comment by dungphanxuan** [comment]
HI, Could you please take a moment to review my pull request for the Vietnamese language translations? Your feedback would be really helpful in ensuring accuracy and quality

File: lang/vi/auth.php [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
```
Comment (devansh-webkul) [inline lang/vi/auth.php:3]: We are not using strict type as of now.

Comment (dungphanx191991) [reply lang/vi/auth.php]: deleted. thank you

File: packages/Webkul/Admin/src/Resources/lang/vi/app.php [hunk 2]
```diff
@@ -0,0 +1,4014 @@
...
+            'price'             => 'Giá',
+            'qty'               => 'Số lượng',
+            'shipping'          => 'Vận chuyển',
+            'shipping-address'  => 'Địa chỉ Giao hàng',
+            'shipping-handling' => 'Xử lý Giao hàng',
+            'sku'               => 'SKU',
+            'subtotal'          => 'Tổng phụ',
+            'tax'               => 'Thuế',
+        ],
+
```
Comment (shivendra-webkul) [inline packages/Webkul/Admin/src/Resources/lang/vi/app.php:4012]: Please remove extra space and match all the arrays some extra space and keys are added.
take the reference from 'en' (english) language. Number of lines should be same.

File: packages/Webkul/Product/src/Resources/lang/vi/app.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        ],
+
+        'bundle'       => 'Gói',
+        'configurable' => 'Có thể cấu hình',
+        'downloadable' => 'Có thể tải xuống',
+        'grouped'      => 'Nhóm',
+        'simple'       => 'Đơn giản',
+        'virtual'      => 'Ảo',
+    ],
+
```
Comment (shivendra-webkul) [inline packages/Webkul/Product/src/Resources/lang/vi/app.php:49]: Remove extra space

File: packages/Webkul/Shop/src/Resources/lang/vi/app.php [hunk 4]
```diff
@@ -0,0 +1,935 @@
...
+            'success'                     => 'Tài khoản đã được tạo thành công.',
+            'success-verify'              => 'Tài khoản đã được tạo thành công, một email đã được gửi để xác minh.',
+            'success-verify-email-unsent' => 'Tài khoản đã được tạo thành công, nhưng email xác minh chưa được gửi.',
+            'verification-not-sent'       => 'Lỗi! Có vấn đề trong việc gửi email xác minh, vui lòng thử lại sau.',
+            'verification-sent'           => 'Email xác minh đã được gửi',
+            'verified'                    => 'Tài khoản của bạn đã được xác minh, hãy thử đăng nhập ngay bây giờ.',
+            'verify-failed'               => 'Chúng tôi không thể xác minh tài khoản email của bạn.',
+        ],
+
+
```
Comment (shivendra-webkul) [inline packages/Webkul/Shop/src/Resources/lang/vi/app.php:71]: Remove extra space

File: packages/Webkul/Shop/src/Resources/lang/vi/app.php [hunk 5]
```diff
@@ -0,0 +1,935 @@
...
+                        'qty'               => 'Số lượng',
+                        'refunds'           => 'Hoàn trả',
+                        'shipping-handling' => 'Vận chuyển & Xử lý',
+                        'sku'               => 'SKU',
+                        'subtotal'          => 'Tổng phụ',
+                        'tax'               => 'Thuế',
+                        'tax-amount'        => 'Số tiền thuế',
+                    ],
+                ],
+
```
Comment (shivendra-webkul) [inline packages/Webkul/Shop/src/Resources/lang/vi/app.php:241]: Also Remove extra space from here.

File: packages/Webkul/Shop/src/Resources/lang/vi/app.php [hunk 6]
```diff
@@ -0,0 +1,935 @@
...
+                'profile'            => 'Hồ sơ',
+                'remove'             => 'Xóa',
+                'remove-all-success' => 'Tất cả các mục trong danh sách mong muốn của bạn đã được xóa',
+                'remove-fail'        => 'Không thể xóa mục khỏi danh sách mong muốn',
+                'removed'            => 'Mục đã được xóa thành công khỏi danh sách mong muốn',
+                'see-details'        => 'Xem chi tiết',
+                'success'            => 'Mục đã được thêm vào danh sách mong muốn thành công',
+                'title'              => 'Danh sách mong muốn',
+            ],
+
```
Comment (shivendra-webkul) [inline packages/Webkul/Shop/src/Resources/lang/vi/app.php:307]: Remove space from here

File: packages/Webkul/Shop/src/Resources/lang/vi/app.php [hunk 7]
```diff
@@ -0,0 +1,935 @@
...
+            'table' => [
+                'actions'              => 'Hành động',
+                'next-page'            => 'Trang kế',
+                'no-records-available' => 'Không có dữ liệu.',
+                'page-navigation'      => 'Điều hướng trang',
+                'page-number'          => 'Số trang',
+                'previous-page'        => 'Trang trước',
+            ],
+
```
Comment (shivendra-webkul) [inline packages/Webkul/Shop/src/Resources/lang/vi/app.php:418]: Remove space from here

File: packages/Webkul/Shop/src/Resources/lang/vi/app.php [hunk 8]
```diff
@@ -0,0 +1,935 @@
...
+            'next'        => 'Tiếp theo',
+            'previous'    => 'Trước đó',
+        ],
+
+        'quantity-changer' => [
+            'decrease-quantity' => 'Giảm Số lượng',
+            'increase-quantity' => 'Tăng Số lượng',
+        ],
+
+        'products' => [
```
Comment (shivendra-webkul) [inline packages/Webkul/Shop/src/Resources/lang/vi/app.php:470]: All arrays and keys should be adjusted to ensure consistency. This array is placed under 'component' it should be outside of component please match all the arrays and keys and remove extra spaces take the reference from 'en' (English) locale.

**Comment by shivendra-webkul** [comment]
@dungphanxuan Also, include Vietnamese locale support in the installer.

File: packages/Webkul/Shop/src/Resources/lang/vi/app.php (hunk 8 above)
Comment (dungphanxuan) [reply packages/Webkul/Shop/src/Resources/lang/vi/app.php:470]: Update 
image

**Comment by dungphanxuan** [comment]
@shivendra-webkul 
Hello, I have updated according to the editing contents, I have checked the settings and succeeded. thank you for reviewing.
image
image

**Comment by devansh-webkul** [comment]
@dungphanxuan,

Please take a pull from the master and check the root lang folder. Your root vitanemese lang file should be match exact same like other locale. Key value pair, lines and spaces should be consistent. You can take reference from the other files.

**Comment by chaunnt** [comment]
@dungphanxuan did you finish your VN translation patch ?

**Pull request closed without merging**